### PR TITLE
Update ELN Extras workload for Meta

### PR DESCRIPTION
**Pull request by davide125**
Add a couple more packages we need.

### CODE REVIEW

File: configs/eln_extras_meta.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ data:
     - duo_unix
     - duo_unix-selinux
     - ebranch
+    - emacs-nw
```
Comment (yselkowitz) [inline configs/eln_extras_meta.yaml:30]: emacs-nw is in ELN.

Comment (davide125) [reply configs/eln_extras_meta.yaml:30]: Is it actually shipped though? `mock -r fedora-eln-x86_64 --install emacs-nw` fails for me here.

Comment (davide125) [reply configs/eln_extras_meta.yaml:30]: I see it listed in https://github.com/minimization/content-resolver-input/blob/main/configs/sst_cs_plumbers-editors.yaml so in theory it should be (and https://tiny.distro.builders/workload--sst_cs_plumbers-editors--fedora-empty-base--repository-fedora-eln--x86_64.html confirms it), but I don't see in https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/AppStream/x86_64/os/Packages/ .

Comment (yselkowitz) [reply configs/eln_extras_meta.yaml:30]: That would need to be fixed in https://pagure.io/pungi-fedora/tree/eln , not in content-resolver.

Comment (davide125) [reply configs/eln_extras_meta.yaml:30]: Pungi is just installing the emacs group, so I think the actual fix for this needs to be in comps: https://pagure.io/fedora-comps/pull-request/950

**Review by yselkowitz — APPROVED** [review]

**Pull request merged**